css/review.css

```diff
@@ -0,0 +1,593 @@
+  background-size: 305px 782px;
+}
+
+.contact-header > .c-header {
+  font-size: 40px;
+  font-weight: bold;
+  padding-top: 100px;
+  color: white;
+  text-align: center;
+  margin: 0;
+}
+
+.contact-header > .description > .c-descrp {
+  color: white;
+  text-align: center;
+  font-size: 20px;
+  padding: 20px;
+}
+
+.contact-us img {
+  padding-top: 15px;
+  max-width: 100%;
+  position: absolute;
+  padding-left: 140px;
+}
+
+form {
+  z-index: 19;
+  padding: 5%;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+}
+
+form input {
+  height: 48px;
+  border-radius: 8px;
+  padding: 0 10px;
+  border: none;
+  margin-bottom: 20px;
+}
+
+form textarea {
+  height: 170px;
+  border-radius: 8px;
+  padding: 10px 10px;
+  border: none;
+  margin-bottom: 20px;
+}
+
+form input[type=submit] {
+  height: 48px;
+  width: 140px;
+  border-radius: 8px;
+  padding: 0 10px;
+  border: none;
+  color: #6070ff;
+  margin-bottom: 20px;
+}
+
+form input[type=submit]:hover {
+  height: 48px;
+  width: 140px;
+  border-radius: 8px;
+  padding: 0 10px;
+  border: 1px solid white;
+  color: white;
+  background: #6070ff;
+  margin-bottom: 20px;
+}
+
+.dash {
+  border-bottom: 6px solid #0d151e;
+  margin: 0 35%;
+  border-radius: 10px;
+}
+
+.mother-accordion li:first-child { border-bottom: 0; }
+
+::-webkit-scrollbar {
+  width: 10px;
+  height: 10px;
+  display: none;
+}
+
+::-webkit-scrollbar-track {
+  box-shadow: inset 0 0 5px grey;
+  border-radius: 10px;
+}
+
+::-webkit-scrollbar-thumb {
+  background: #6070ff;
+  border-radius: 10px;
+}
+
+::-webkit-scrollbar-thumb:hover {
+  background: #3847d4;
+}
+
+@media (max-width: 768px) {
+  nav .button {
+    display: block;
+  }
+
+  nav .destop-menu {
+    display: none;
+  }
+
+  .dash {
+    display: none;
+  }
+}
+
+@media (min-width: 768px) {
+  .my-work {
+    flex-direction: row;
+    gap: 50px;
+  }
+
+  .my-work img,
+  .my-work-body {
+    width: 100%;
+  }
+
+  .my-work:nth-child(2n) {
+    flex-direction: row-reverse;
+  }
+
+  .tech {
+    margin-top: 30px;
+    margin-bottom: 50px;
+  }
+
+  .my-work p {
+    margin-top: 15px;
+  }
+
+  .position-container ul {
+    justify-content: flex-start;
+  }
+
+  .position-container li {
+    margin-right: 50px;
+  }
+
+  .section3 {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    gap: 50px;
+    align-items: center;
+  }
+
+  .about,
+  .accordion {
+    width: 100%;
+  }
+
+  .child-accordion {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
+  }
+
+  .child-accordion li {
+    margin-right: 10px;
+  }
+
+  .language-name {
+    flex-direction: column;
+  }
+
+  .language-name span {
+    align-items: left;
+    width: 100%;
+    padding: 10px;
+  }
+
+  .background-image {
+    background-image: url('../image/contact-pc-bg.svg');
+    background-repeat: no-repeat;
+    background-size: 1135px 720px;
+    background-position: center;
```

Comment:
kindly use % for background instead of px for better scaling.

Reply:
Kindly add text in the alt attribute in order to increase accessibility on the page

Reply:
@AbhishekkM05 Great Observation.